package/manage.sh

```diff
@@ -60,7 +60,7 @@ tailscale_install() {
   TAILSCALE_TGZ="${WORKDIR}/tailscale.tgz"
 
   echo "Installing Tailscale v${VERSION} in ${TAILSCALE_ROOT}..."
-  curl -sSL -o "${TAILSCALE_TGZ}" "https://pkgs.tailscale.com/stable/tailscale_${VERSION}_arm64.tgz"
+  curl -sSLf -o "${TAILSCALE_TGZ}" "https://pkgs.tailscale.com/stable/tailscale_${VERSION}_arm64.tgz"
```

Comment:
Thank you, yeah that's not a bad idea - I'm also looking to see if there's any version source that I can consume to get a sense of what is released for specific architectures (instead of checking GitHub Releases) so that we don't run into this same issue again in future.

Reply:
Good news, it looks like there's an official source which should solve this problem. I'm going to get this merged and a new release out to solve the problem that folks are running into.